portfolio/src/main/webapp/genre_constants.js

```diff
@@ -0,0 +1,50 @@
+const postPunk = ["Punks - Matt Champion", "Fangs - matt champion", "Call this # now - the garden", "Stylish spit - the garden", "All access - the garden", "Clench to stay awake - the garden", "Bizarre love triangle - new order", "Neat neat neat - the damned", "Emergency blimp - king krule", "Vidual - king krule", "Half man half shark - king krule", "Comet face - king krule", 
+    "Blue train lines - mount kimbie", "Midnight 01 (deep sea diver) - king krule", "Heads will roll - yeah yeah yeahs", "Wolf like me - tv on the radio", "Tenement roofs - glorious din", "Big joanie - dream no 9", "Rip it up - orange juice", "Blister in the sun - violent femmes", "Disorder - joy division"];
+const hipHopIds = ["HdOhwPovkHI", "CXdTShfFPYM", "Wzurw1w3AjY", "FUXX55WqYZs", "iOtRxcTMsHc", "XXMHQ410P-c", "t5gh4uX-X40",
+    "0yCJs3GrjpE", "F-4KiszXxHM", "4cucHc2vef4", "aIXyKmElvv8", "F2r-8fd1jVE", "AC4bb9Q9-04", "tYVebHOrlZU", "f--ZuvER98U",
+    "8KxXsQjJsIw", "zI19cWqsLhA", "lDWeob50YY8", "sTTU-T9-RTs", "OJyy_stKS9c", "1og6jfhUPxE"];
+const rbIds = ["lnUV4cz0gv8", "OxdCR6wwoXs", "lovts-q5s0s", "vT02Lo5WhLI", "YzI76YfVXD4", "kobOjAkSrhk", "pvU4b4N1-QU", 
+    "TftBcX-kCQc", "9G-vhg8T2rc", "DBkjjuAoajY", "7HiB-hNt_mw", "6Fc4QwsTz_g", "i9PmsucjHrM", "NclTIViKiyE", "6UpjbnDfbRU",
+    "3xotpsdLS6c", "dta4heZacZE", "opRwZJsOe0w", "uzS3WG6__G4", "HK90QhbzIkw", "jEvv2PR0ZeI"];
+const funkIds = ["ftdZ363R9kQ", "S6Y1gohk5-A", "l7vRSu_wsNc", "MhC_iLBrSUY", "HhPu6GEoad8", "ALzISTfdXpQ", "vuDCi5neNMM",
+    "NOa5UOHdwnc", "T1NW57lk5fY", "50W9W0RHvwI", "yURRmWtbTbo", "6To0fvX_wFA", "EnNgASBdCeo", "oTUSeac7IuQ", "C02SFbc4zmM",
+    "42YGprrAOj0", "qchPLaiKocI", "JTvlujbJ5vg", "5SrQ9graFWQ", "71l85z2bXAs", "eAk3ayxi8Gk"];
+const bedroomPopIds = ["Mf5XrSuQ4R8", "2p7m2iY-WSY", "A0gpfwZP_To", "WMp0ZyQWKf4", "GYEKlxUty84", "dvWZt6SuNtI",
+    "zC3GHcPN4EQ", "sTQNJT7OZew", "L9HYJbe9Y18", "L9l8zCOwEII", "SRbzuwYMyDE", "QkeehQ8D_VM", "oFYOxVeQBTo", "GM8YDU6ZjBw",
+    "7VVLDCR4wPM", "xqC--4paOTM", "5ohMAVDUswQ", "qTe6toWNC6s", "OeCS-DZsgKM", "jCkBxmPbohk", "WMp0ZyQWKf4"];
+const neoSoulIds = ["6giKIu5jUvA", "T6QKqFPRZSA", "_W14wK4QGh4", "D7rm9t5S4uE", "0a4PD4SwEAs", "0ZARYxRmNWg", "OQuQV6u8-tk",
+    "2j4qvVyRgRk", "Gb76TgCUqAY", "t-E2gm0a_N0", "mXiFHDfvn4A", "wVzvXW9bo5U", "hToe5L41y6c", "uhjtjOUsQ6c", "2w6udgiojlE",
+    "ajOCGskHs2A", "ygkIMT4vsTE", "n0tWINt8Ofk", "T3CunfPYkME", "SxVNOnPyvIU", "ISCFeQ7963A"];
+const postPunkIds =["6qHClRnSQYM", "l03ybrIMxKo", "JRiSdAe8sGY", "rW43Sq4Rgjo", "c7rsPaDXtng", "N5an4_Iw7ZU", "tkOr12AQpnU",
+    "VaMU_6sI3j8", "iAmZPxuMHsc", "kfFhOsjz7jc", "2XzXLzA2Hrc", "k-lg3v4HtBA", "OMMlPs0O2rw", "Ncq7I-EHQ0Q", "PCZ2Dp6Is9M",
+    "j1-xRk6llh4", "XV765LFwHJE", "uIYo1okOm9U", "UzPh89tD5pA", "ImIESKQKKJo", "5BIElTtN6Fs"];
+
+// Populate maps
+const genreToListMap = new Map();
+const genreToIdList = new Map();
+
+genreToListMap.set('hip-hop', hipHop);
```

Comment:
To initialize a map with multiple values at a time:

let recipeMap = new Map([
  ['cucumber', 500],
  ['tomatoes', 350],
  ['onion',    50]
]);